drivers/usbdev_synopsys_dwc2/usbdev_synopsys_dwc2.c

```diff
@@ -749,10 +807,53 @@ static void _usbdev_init(usbdev_t *dev)
     /* USB VBUSEN pin is not yet used */
     /* USB_ROUTELOC0 = location */
 
+#elif defined(MCU_GD32V)
+
+    /* Block both DEEP_SLEEP and STANDBY, TODO DEEP_SLEEP is unblocked during
+     * USB suspend status */
+    pm_block(GD32V_PM_DEEPSLEEP);
+    pm_block(GD32V_PM_STANDBY);
+
+    /* ensure that PLL clock is used */
+    assert((RCU->CFG0 & RCU_CFG0_SCS_Msk) == RCU_CFG0_SCS_PLL);
+    /* ensure that the 48 MHz USB clock can be generated */
+    assert((CLOCK_CORECLOCK == MHZ(48)) || (CLOCK_CORECLOCK == MHZ(72)) ||
+           (CLOCK_CORECLOCK == MHZ(96)) || (CLOCK_CORECLOCK == MHZ(120)));
```

Comment:
Could this be changed into a `static_assert`?

Reply:
Yep, changed.